crates/bevy_hierarchy/src/child_builder.rs

```diff
@@ -279,39 +279,62 @@ pub struct ChildBuilder<'a> {
     push_children: PushChildren,
 }
 
-impl ChildBuilder<'_> {
+/// Trait for building children entities and adding them to a parent entity. See
```

Comment:
The "see" comment is unclear to readers. You should explain how these are related.